ttnn/cpp/ttnn/operations/ccl/ccl_common.cpp

```diff
@@ -90,13 +90,13 @@ KernelHandle generate_edm_kernel(
         log_trace(tt::LogOp, "\t{}", s);
     }
 
-    auto eth_sender_kernel = tt_metal::CreateKernel(
+    auto eth_sender_kernel =tt::tt_metal::CreateKernel(
         program,
-        "tt_eager/tt_dnn/op_library/ccl/edm/erisc_datamover.cpp",
+        "ttnn/cpp/ttnn/operations/ccl/edm/erisc_datamover.cpp",
```

Comment:
kernels into a kernels folder please `ttnn/cpp/ttnn/operations/ccl/kernels/edm/erisc_datamover.cpp`

Reply:
@ayerofieiev-tt I have moved the files to kernels directory